lib/mengpaneel/tracker.rb

```diff
@@ -90,6 +90,15 @@ def update(message)
 
         super(message)
       end
+
+      # mixpanel-ruby only handles hash, whereas Javascript handles string and hash.
+      def increment(*args)
+        if args.first.is_a?(Hash)
+          super(tracker.distinct_id, *args)
```

Comment:
We need the `unless args.first == tracker.distinct_id` check, because methods like `plus_one` call `increment` internally.